src/components/TimetablePage/DefaultPage/index.tsx

```diff
@@ -0,0 +1,387 @@
+    token
+      ? (myLecturesFromServer ?? [])
+      : (myLecturesFromLocalStorageValue ?? []),
+  );
+
+  const selectedLecture = useRecoilValue(selectedTempLectureSelector);
+  const { data: lectureList, status } = useLectureList(selectedSemester);
+  const similarSelectedLecture = (lectureList as unknown as Array<LectureInfo>)
+    ?.filter((lecture) => lecture.code === selectedLecture?.code)
+    ?? [];
+  const similarSelectedLectureDayList = useTimetableDayList(similarSelectedLecture);
+  const selectedLectureIndex = similarSelectedLecture
+    .findIndex(({ lecture_class }) => lecture_class === selectedLecture?.lecture_class);
+
+  return selectedSemesterValue && status === 'success' ? (
+    <Timetable
+      lectures={myLectureDayValue}
+      similarSelectedLecture={similarSelectedLectureDayList}
+      selectedLectureIndex={selectedLectureIndex}
+      colWidth={55}
+      firstColWidth={52}
+      rowHeight={21}
+      totalHeight={456}
+    />
+  ) : (
+    <div>
+      loading...
+    </div>
+  );
+}
+
+function Curriculum() {
+  const { data: deptList } = useDeptList();
+  return (
+    <ul className={styles['page__curriculum-list']}>
+      {(deptList as unknown as Array<IDept> | undefined ?? []).map((dept) => (
+        <li key={dept.name}>
+          <a
+            className={styles.page__curriculum}
+            href={dept.curriculum_link}
+          >
+            {dept.name}
+          </a>
+        </li>
+      ))}
+      <li>
+        <a
+          className={styles.page__curriculum}
+          href="https://www.koreatech.ac.kr/kor/sub01_03_02.do"
+        >
+          대학 입학 요람
+        </a>
+      </li>
+    </ul>
+  );
+}
+
+function DefaultPage() {
+  const {
+    searchInputRef,
+    onClickSearchButton,
+    onKeyDownSearchInput,
+    // value: searchValue,
+  } = useSearch();
+  const {
+    value: deptFilterValue,
+    onChangeSelect: onChangeDeptSelect,
+  } = useSelect();
+  const {
+    value: semesterFilterValue,
+    onChangeSelect: onChangeSemesterSelect,
+  } = useSelectRecoil(selectedSemesterAtom);
+
+  return (
+    <>
+      <h1 className={styles.page__title}>시간표</h1>
+      <div className={styles.page__content}>
+        <div>
+          <div className={styles.page__filter}>
+            <div className={styles['search-input']}>
+              <input
+                ref={searchInputRef}
+                className={styles['search-input__input']}
+                placeholder="교과명을 입력하세요."
+                onKeyDown={onKeyDownSearchInput}
+              />
+              <button
+                className={styles['search-input__button']}
+                type="button"
+                onClick={onClickSearchButton}
+              >
+                <img src="https://static.koreatech.in/assets/img/ic-search-gray.png" alt="search" />
+              </button>
+            </div>
+            <div className={styles.page__depart}>
+              <React.Suspense fallback="loading...">
```

Comment:
Bus쪽 코드에서는 `LoadingSpinner` svg파일을 활용해 fallback component를 구성했는데, 활용해보는건 어떨까요?
```ts
import { ReactComponent as LoadingSpinner } from 'assets/svg/loading-spinner.svg';
```
```suggestion
              <React.Suspense fallback={<LoadingSpinner />}>
```

Reply:
해당 부분에는 스피너보다는 스켈레톤이 어울릴 것 같아 이렇게 작성하게 되었습니다. 이후 구현을 목표로 두는 것이 좋을 것으로 보입니다.